pkg/controllers/managementuser/systemcharts/systemcharts.go

```diff
@@ -0,0 +1,88 @@
+// Package systemcharts hold a controller used to managed systemcharts in the UserContext.
+package systemcharts
+
+import (
+	"context"
+	"fmt"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/controllers/dashboard/chart"
+	"github.com/rancher/rancher/pkg/namespace"
+	"github.com/rancher/rancher/pkg/types/config"
+	wcorev1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var configHandlerName = "systemcharts-config-sync"
+
+type controller struct {
+	dsConfigClient wcorev1.ConfigMapClient
+	dsConfigCache  wcorev1.ConfigMapCache
+	clusterName    string
+}
+
+// Register registers the handlers needed for the systemcharts controller.
+func Register(ctx context.Context, cluster *config.UserContext) {
+	ctrl := controller{
+		clusterName:    cluster.ClusterName,
+		dsConfigClient: cluster.Corew.ConfigMap(),
+		dsConfigCache:  cluster.Corew.ConfigMap().Cache(),
+	}
+	cluster.Management.Wrangler.Mgmt.Cluster().OnChange(ctx, configHandlerName, ctrl.onClusterChange)
+}
+
+// onClusterChange handler watches for changes to the v3.Cluster in the local cluster and syncs the updated configMap to the downstream cluster.
+func (c *controller) onClusterChange(_ string, cluster *v3.Cluster) (*v3.Cluster, error) {
+	if cluster == nil || cluster.Name != c.clusterName || cluster.Spec.ChartValues == nil {
+		return cluster, nil
+	}
+	webhookValues := cluster.Spec.ChartValues[chart.WebhookChartName]
+
+	// using a cache here means we are indexing all of the configMaps on all downstream clusters.
+	// currently the indexer is already in use for the snapshotbackpopulate controller.
+	// if that controller is ever removed this can be switched to use a client for reduced memory usage.
+	clusterConfigMap, err := c.dsConfigCache.Get(namespace.System, chart.CustomValueMapName)
```

Comment:
Near as I can tell, the [snapshotbackpopulate](https://github.com/rancher/rancher/blob/5f98b0f28bc8a48366e9ced8f0a0fea3d826d90d/pkg/controllers/managementuser/snapshotbackpopulate/snapshotbackpopulate.go#L70) uses a norman cache/controller for this value rather than the wrangler one that you added for use here. Does adding the wrangler controller increase memory use, or are the backing objects shared between the norman/wrangler controllers?